source/includes/_feature-deriving.md

```diff
@@ -395,4 +395,158 @@ POST'ing to the private variables catalog a Shoji Entity containing a ZCL functi
 
 ### Other transformations
 
-Forthcoming.
+#### bin
+Receives a numeric column and returns a categorical column where each
+category represents a bin of the numeric values.
+Each category is annotated with a "boundaries" member that contains the 
+lower/upper bound of each bin.
+
+```json
+{
+    "function": "bin",
+    "args": [
+        {"variable": "http://.."}
+    ]
+}
+```
+
+#### case
+Returns a categorical column with its categories following the specified
+conditions from different columns on the dataset. [View Case Statements](#Case-statements)
+
+#### cast
+Returns a new column with its type and values casted. Not applicable
+on arrays or date columns.
+
+```json
+{
+    "function": "cast",
+    "args": [
+        {"variable": "http://.."},
+        {"value": "numeric"}
+    ]
+}
+```
+
+The allowed types are:
+
+* numeric
+* text
+* categorical
+
+For categorical types it is necessary to indicate the categories as a type
+definition.
+
+```json
+{
+    "function": "cast",
+    "args": [
+        {"variable": "http://.."},
+        {"value": {
+                "class": "categorical",
```

Comment:
I think it is necessary as a way to go back and forth? I'd gladly remove it but seemed incomplete to not include one very evident type. 

I am rephrasing the line above the example.